src/components/box/box.test.js

```diff
@@ -0,0 +1,666 @@
+      }
+    );
+
+    it.each([
+      [400, "400px"],
+      [800, "600px"],
+    ])(
+      "should verify when Height is set to %s that Box height is not more than maxHeight %s",
+      (height, maxHeight) => {
+        CypressMountWithProviders(
+          <BoxComponent maxHeight={600} height={height} />
+        );
+
+        getDataElementByValue("box").should("have.css", "height", maxHeight);
+      }
+    );
+
+    it.each([
+      ["400px", "400px"],
+      ["800px", "600px"],
+    ])(
+      "should verify when Height is set to %s that Box height is not more than maxHeight %s",
+      (height, maxHeight) => {
+        CypressMountWithProviders(
+          <BoxComponent maxHeight={600} height={height} />
+        );
+
+        getDataElementByValue("box").should("have.css", "height", maxHeight);
+      }
+    );
+
+    it.each(["normal", "stretch", "baseline"])(
+      "should verify Box alignItmes is %s",
+      (alignment) => {
+        CypressMountWithProviders(<BoxComponentMulti alignItems={alignment} />);
+
+        getDataElementByValue("box").should(
+          "have.css",
+          "align-items",
+          alignment
+        );
+      }
+    );
+
+    it.each(["normal", "baseline"])(
+      "should verify Box alignContent is %s",
+      (alignment) => {
+        CypressMountWithProviders(
+          <BoxComponentMulti alignContent={alignment} />
+        );
+
+        getDataElementByValue("box").should(
+          "have.css",
+          "align-content",
+          alignment
+        );
+      }
+    );
+
+    it.each(["left", "center", "right"])(
+      "should verify Box justifyContent is %s",
+      (justified) => {
+        CypressMountWithProviders(
+          <BoxComponentMulti justifyContent={justified} />
+        );
+
+        getDataElementByValue("box").should(
+          "have.css",
+          "justify-content",
+          justified
+        );
+      }
+    );
+
+    it.each(["nowrap", "wrap", "wrap-reverse"])(
+      "should verify Box flex wrap is %s",
+      (wrap) => {
+        CypressMountWithProviders(<BoxComponentMulti flexWrap={wrap} />);
+
+        getDataElementByValue("box").should("have.css", "flex-wrap", wrap);
+      }
+    );
+
+    it.each(["column", "column-reverse", "row", "row-reverse"])(
+      "should verify Box flex direction is %s",
+      (direction) => {
+        CypressMountWithProviders(
+          <BoxComponentMulti flexDirection={direction} />
+        );
+
+        getDataElementByValue("boxone").should(
+          "have.css",
+          "flex-direction",
+          direction
+        );
+      }
+    );
+
+    it.each(["auto", "content", "fit-content", "max-content", "min-content"])(
+      "should verify Box flex basis is %s",
+      (flex) => {
+        CypressMountWithProviders(<BoxComponentMulti flex={flex} />);
+
+        getDataElementByValue("box").should("have.css", "flex-basis", flex);
+      }
+    );
```

Comment:
```suggestion
```
This seems to be a duplicate of the flex basis of the test on line 616. 

I appreciate the flex properties are not easy since they can take several values! The `flex` css property is a shorthand of `flex-grow`, `flex-shrink` and `flex-basis` together, so I'm not sure if need a test for `flex` since its functionality is covered by the other tests anyway, or if we should test it since it's still a prop we are still providing to consumers.